src/WriteModels/OrderItemWrite.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace VaniloCloud\WriteModels;
+
+use VaniloCloud\Enums\ProductType;
+
+class OrderItemWrite extends BaseWriteModel
+{
+    public function setName(string $name): self
+    {
+        $this->data['name'] = $name;
+
+        return $this;
+    }
+
+    public function setSku(?string $sku): self
+    {
+        $this->data['sku'] = $sku;
+
+        return $this;
+    }
+
+    public function setProductId(?string $product_id): self
+    {
+        $this->data['product_id'] = $product_id;
+
+        return $this;
+    }
+
+    public function setProductType(?ProductType $product_type): self
+    {
+        $this->data['product_type'] = $product_type;
+
+        return $this;
+    }
+
+    public function setPrice(float $price): self
+    {
+        $this->data['price'] = $price;
+
+        return $this;
+    }
+
+    public function setOriginalPrice(?float $original_price): self
+    {
+        $this->data['original_price'] = $original_price;
+
+        return $this;
+    }
+
+    public function setQuantity(?int $quantity): self
+    {
+        $this->data['quantity'] = $quantity;
+
+        return $this;
+    }
+
+    public function set__Instructions(array $__instructions): self
```

Comment:
The method name `set__Instructions` violates the [PSR-1][1] and [PSR-12][2] coding style guides, which define best practices for method naming in PHP:

1. **PSR-1 (Basic Coding Standard)**:
   - Method names SHOULD be declared in `camelCase`.
   - Using double underscores `__` is unusual in method names, and PSR does not recommend such use. Double underscores are typically reserved for magic methods (e.g., `__construct`, `__get`).
2. **PSR-12 (Extended Coding Style)**:
   - This standard extends PSR-1 and further enforces the `camelCase` method naming convention.
   - There’s no scenario in PSR-12 where `set__Instructions` would be considered acceptable because of the double underscore and mixed case usage.

So, a more PSR-compliant version would be something like `setInstructions`.

### TL;DR:

To follow the PSR guidelines, rename `set__Instructions` to `setInstructions` or another camelCase variant without unnecessary underscores.

[1]: https://www.php-fig.org/psr/psr-1/#43-methods
[2]: https://www.php-fig.org/psr/psr-12/#44-methods-and-functions

Reply:
Also the parameter name shouldn't start with `__` underscores. It's a good thing that the schema is being copied over from the OpenAPI spec, but we also need to adhere with the PHP coding standards, so as a generic rule, adjustment to PHP naming is desired.